### PR TITLE
Complete section for each project

### PR DESCRIPTION
**Pull request by remusne**
Add description and GitHub link for each project,

### CODE REVIEW

File: portfolio/src/main/webapp/projects.html [hunk 1]
```diff
@@ -9,36 +9,104 @@
...
    
     <section id="firstProject">
         <h1 class ="project">WorldWide Knowledge</h1>
         <a href="#dashboard">Go back to dashboard</a>
         <a href="#secondProject">Go to next project</a>
+
+        <div class="content">
+            <h4><b>Description:</b></h4>
+            <ul>
+                <li>Android application used forimproving geography skills
```
Comment (semenovm) [inline portfolio/src/main/webapp/projects.html]: for improving

**Review by semenovm — DISMISSED** [review]

File: portfolio/src/main/webapp/projects.html [hunk 2]
```diff
@@ -9,36 +9,104 @@
...
         <h1 class ="project">Projects</h1>
-        <a href="#firstProject">Go to first project</a>
-        <a href="#secondProject">Go to second project</a>
-        <a href="#thirdProject">Go to third project</a>
+
+        <div id="projectList">
+            <a href="#firstProject">
+                <div class="thumbnail"> 
+                    <img src="earth.png" alt="WWK logo">
+                    <h1 class="projectTitle">WorldWide Knowledge</h1>
```
Comment (maxwbuckley) [inline portfolio/src/main/webapp/projects.html]: Worldwide

File: portfolio/src/main/webapp/projects.html [hunk 3]
```diff
@@ -9,36 +9,104 @@
...
    
     <section id="firstProject">
         <h1 class ="project">WorldWide Knowledge</h1>
         <a href="#dashboard">Go back to dashboard</a>
         <a href="#secondProject">Go to next project</a>
+
+        <div class="content">
+            <h4><b>Description:</b></h4>
+            <ul>
+                <li>Android application used for improving geography skills
```
Comment (maxwbuckley) [inline portfolio/src/main/webapp/projects.html]: s/geography skills/geographic knowledge/

File: portfolio/src/main/webapp/projects.html [hunk 4]
```diff
@@ -9,36 +9,104 @@
...
     <section id="secondProject">
         <h1 class ="project">Ride Sharing Service Simulator</h1>
         <a href="#dashboard">Go back to dashboard</a>
         <a href="#firstProject">Go to previous project</a>
         <a href="#thirdProject">Go to next project</a>
+
+        <div class="content">
+            <h4><b>Description:</b></h4>
+            <ul>
+                <li>Linux C++ application that assignes drivers for each client
```
Comment (maxwbuckley) [inline portfolio/src/main/webapp/projects.html]: assigns

File: portfolio/src/main/webapp/projects.html [hunk 5]
```diff
@@ -9,36 +9,104 @@
...
         <a href="#secondProject">Go to previous project</a>
+
+        <div class="content">
+            <div>
+                <h4><b>Description:</b></h4>
+                <ul>
+                    <li>Arduino-based project that measures the
+                        velocity using a Hall magnetic-sensor.</li>
+                    <li>Won the bronze medal at the UPB Science Fair 2019.</li>
+                    <li>The algorithms minimize the average cost and duration for a ride.</li>
```
Comment (maxwbuckley) [inline portfolio/src/main/webapp/projects.html]: Should this sentence be here?

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -79,3 +81,73 @@ a {
...
+}
+
+.gitHub {
+    margin-top: 50px;
+    color:  #000066;;
+    text-decoration: none;
+    font-size:30px;
+    left: 10vw;
+    position: relative;
+}
```
Comment (maxwbuckley) [inline portfolio/src/main/webapp/style.css]: New line at the end

**Review by maxwbuckley — APPROVED** [review]

File: portfolio/src/main/webapp/projects.html (hunk 5 above)
Comment (remusne) [reply portfolio/src/main/webapp/projects.html]: My bad, sorry.

**Pull request merged**